React/Modules/RCTDevMenu.m

```diff
@@ -514,6 +516,19 @@ - (NSString *)devMenuSettingsKey {
     return;
   }
 
+  BOOL inBuildMode = _bundleURL.fileURL == YES;
```

Comment:
Don't need the `== YES`, just assign to the `.fileURL` value. 
http://stackoverflow.com/a/1934595/1144632